my_tiny_service/api/api.py

```diff
@@ -1,3 +1,4 @@
+from .timestamp import router as timestamp_router
```

Comment:
@acedev-ai let's use the global import for `timestamp_router`

Reply:
@acedev-ai nice! let's also remove the `from .timestamp import router as timestamp_router` import. I think it's invalid